Configs/.config/hypr/hyprlock/presets/layout3.conf

```diff
@@ -0,0 +1,103 @@
+#      ░▒▒▒░░░░░▓▓          ___________
+#    ░░▒▒▒░░░░░▓▓        //___________/
+#   ░░▒▒▒░░░░░▓▓     _   _ _    _ _____
+#   ░░▒▒░░░░░▓▓▓▓▓▓ | | | | |  | |  __/
+#    ░▒▒░░░░▓▓   ▓▓ | |_| | |_/ /| |___
+#     ░▒▒░░▓▓   ▓▓   \__  |____/ |____/
+#       ░▒▓▓   ▓▓  //____/
+
+$hyprlockDir = $HOME/.config/hypr/hyprlock
+$wallpaper = $HOME/.cache/hyde/wall.set
+$fontFamily = JetBrainsMono Nerd Font
+source = $hyprlockDir/colors.conf
+
+background {
+    monitor =
+    path = $wallpaper
+    blur_size = 4
+    blur_passes = 3 # 0 disables blurring
+    noise = 0.0117
+    contrast = 1.3000 # Vibrant!!!
+    brightness = 0.8000
+    vibrancy = 0.2100
+    vibrancy_darkness = 0.0
+}
+
+# DAY
+label {
+    monitor =
+    text = cmd[update:1000] echo "$(date +"%A" | tr '[:lower:]' '[:upper:]')"  
```

Comment:
```suggestion
   text = cmd[update:1000] echo "$(date +"%A" | tr '[:lower:]' '[:upper:]' | sed 's/ID/I D/g')"
```

before :
![241206_13h17m53s_screenshot](https://github.com/user-attachments/assets/2adb26e0-d2c7-4639-bf65-a50f3daac80e)

after :
![241206_13h16m51s_screenshot](https://github.com/user-attachments/assets/4dd27f27-459d-478a-aa7c-fcdf74f875ee)

friday has this wierd text where the letters I and D are blending in looks like "FRDAY" lol

Reply:
i thought it was a font style thing. thanks